src/extension.ts

```diff
@@ -1180,17 +1181,25 @@ function getPorts() {
 async function describeKubernetes(explorerNode?: explorer.ResourceNode) {
     if (explorerNode) {
         const nsarg = explorerNode.namespace ? `--namespace ${explorerNode.namespace}` : '';
-        const result = await kubectl.invokeAsync(`describe ${explorerNode.resourceId} ${nsarg}`);
+        const cmd = `describe ${explorerNode.resourceId} ${nsarg}`;
+        const result = await kubectl.invokeAsync(cmd);
+        const refresh = (): Promise<ShellResult | undefined> => {
```

Comment:
This syntax looks a bit odd to me.  Should it not be `const refresh = () => { … }` or `const refresh: () => Promise<ShellResult | undefined> = () => { … }`?  The way it's laid out looks like the Promise is a type annotation on `()`.  Or perhaps this is another TypeScript idiosyncrasy I don't yet know!

Reply:
What I was trying to say was:

"refresh is a function with no parameters which returns a Promise<...>" and I _think_ that's what Typescript sees... Happy to adjust if you want.